.github/workflows/dev-image-update.yaml

```diff
@@ -76,51 +76,30 @@ jobs:
     needs:
       - bump_version
     steps:
-      - name: Checkout Develop branch of jade-data-repo
-        uses: actions/checkout@v3
-        with:
-          ref: develop
-          token: ${{ secrets.BROADBOT_TOKEN }}
-      - name: 'Checkout datarepo-helm-definitions repo'
+      - name: Checkout tagged branch of jade-data-repo
         uses: actions/checkout@v3
         with:
-          repository: 'broadinstitute/datarepo-helm-definitions'
+          ref: ${{ needs.bump_version.outputs.api_image_tag }}
           token: ${{ secrets.BROADBOT_TOKEN }}
```

Comment:
Future opportunity: does checkout need the broadbot token?

Reply:
Future opportunity: it looks like our bumper does call into main first, yuck. Can we pull it out somewhere to remove our dependency on `datarepo-actions`?

Reply:
Good idea, looks like we already have - I updated https://broadworkbench.atlassian.net/browse/DCJ-658 to encompass this suggestion!